components/DpLabel/DpLabel.vue

```diff
@@ -23,9 +23,9 @@
 </template>
 
 <script>
-import { CleanHtml } from '../../directives'
+import { CleanHtml } from '../../directives/CleanHtml/CleanHtml'
```

Comment:
do we want to import like this here or like `import { CleanHtml } from '../../'` ?
I see both versions in this PR

Reply:
i like the '../../' more. even more i'd like some `@components` / `@directives` alias but thats a thing for another day.